shuup/front/apps/personal_order_history/urls.py

```diff
@@ -11,8 +11,8 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^orders/$', views.OrderListView.as_view(),
+    url(r'^order-history/$', views.OrderListView.as_view(),
```

Comment:
This change is probably breaking some hard coded notification scrips in some instances. Should be included in changelog.